realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -932,6 +932,15 @@ private boolean containsDot(String s) {
         return this;
     }
 
+    /**
+     * Negate condition.
+     * @return The query object
+     * @throws java.lang.RuntimeException Any error
```

Comment:
Reading the code of `CATCH_STD()` it seems that `RuntimeError`, `OutOfMemory`, and `Unspecified` can be thrown. This is true for most `RealmQuery` methods.